.github/workflows/build.yml

```diff
@@ -56,54 +51,44 @@ jobs:
         convert_osx: false
         test_pyver: 3.9
         test_npver: 1.22
-    - name: Upload artifact of the conda package.
+    - name: Upload artifact of the conda package
       uses: actions/upload-artifact@v3
       with:
         name: cil-package
         path: recipe/linux-64/cil*
-  docs_build:
-    needs: conda_build
+  docs:
+    needs: conda
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
     - run: ls
-    - name: Download artifact of the conda package.
+    - name: Download artifact of the conda package
       uses: actions/download-artifact@v3
       with:
         name: cil-package
-        path: 'conda_package'
-    - uses: conda-incubator/setup-miniconda@v2
+        path: conda_package
+    - uses: conda-incubator/setup-miniconda@v3
       with:
         python-version: 3.9
     - uses: lauramurgatroyd/build-sphinx-action@v0.1.4
       with:
-        DOCS_PATH: docs
-        CONDA_BUILD_ENV_FILEPATH: docs/docs_environment.yml
-        ARTIFACT_NAME: DocumentationHTML
-        PACKAGE_FOLDER_PATH: conda_package
-        PACKAGE_NAME: cil
-        PACKAGE_CONDA_CHANNELS: conda-forge -c intel -c ccpi
-        BUILD_SUBDIR_NAME: nightly
+        docs_path: docs
+        conda_build_env_filepath: docs/docs_environment.yml
+        artifact_name: DocumentationHTML
+        package_folder_path: conda_package
+        package_name: cil
+        package_conda_channels: conda-forge -c intel -c ccpi
+        build_subdir_name: nightly
         python_version: 3.9
-  docs_publish:
-    needs: docs_build
-    if: github.ref == 'refs/heads/master'
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - name: Download artifact of the html output.
-      uses: actions/download-artifact@v3
-      with:
-        name: DocumentationHTML
-        path: docs/build
```

Comment:
without this, are the docs in the path docs/build, which is later (optionally) pushed to the gh-pages branch?

Reply:
Yes